RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByMVAIsolationRun2.cc

```diff
@@ -161,7 +161,6 @@ class PFRecoTauDiscriminationByMVAIsolationRun2 : public PFTauDiscriminationProd
   std::unique_ptr<PFTauDiscriminator> category_output_;
```

Comment:
the enums defined on L141 are now somewhat redundant with what's defined in the included PFRecoTauClusterVariables.h.
There is no obvious conflict here due to still fully localized use of these values in mvaOpt_.
So, cleanup is not required, but could still be better.